resources/js/Pages/Pets/Partials/PetDetails.vue

```diff
@@ -0,0 +1,266 @@
+<script setup>
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { getPetTags } from '../../../data/petTagsConfig.js'
```

Comment:
You can use @ it's a root path alias so you don't need to use relative path segments

```suggestion
import { getPetTags } from '@/data/petTagsConfig.js'
```